mollie/onboarding.go

```diff
@@ -62,22 +62,24 @@ func (os *OnboardingService) GetOnboardingStatus(ctx context.Context) (res *Resp
 // at least one of them needs to be provided in the request.
 //
 // Information that the merchant has entered in their dashboard will not be overwritten.
+type OnboardingDataOrganization struct {
+	Name               string   `json:"name,omitempty"`
+	Address            *Address `json:"address,omitempty"`
+	RegistrationNumber string   `json:"registrationNumber,omitempty"`
+	VatNumber          string   `json:"vatNumber,omitempty"`
+	VatRegulation      string   `json:"vatRegulation,omitempty"`
+}
+type OnboardingDataProfile struct {
```

Comment:
A comment for each struct must be provided describing the purpose and content of it.

What I usually do is try to use the "docs" phrasing and just make tiny changes to fit the implementation.